pytorch_lightning/metrics/classification.py

```diff
@@ -133,8 +129,15 @@ def forward(self, pred: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
         Return:
             A Tensor with the confusion matrix.
         """
-        return confusion_matrix(pred=pred, target=target,
-                                normalize=self.normalize)
+        return confusion_matrix(pred=pred, target=target, normalize=self.normalize)
+
+    def aggregate(self, *tensors: torch.Tensor) -> torch.Tensor:
+        """Aggregates results by stacking them instead of concatenating before averaging.
```

Comment:
only works when all tensors have the same shape, but that's probably the case, right?

Reply:
That is an assumption we make and for almost every case it holds. If it doesn't, you have to overwrite this method.